main.go

```diff
@@ -159,23 +161,34 @@ func (self *PostgresLogParser) parseLogBuffer() (*PostgresLogLine, error) {
 	timestamp := parseTime(self.buffer)
 	user, database := parseUserAndDatabase(self.buffer)
 	logType, statementName := parseLogTypeWithStatementName(self.buffer)
-	value := parseValueFromBuffer(self.buffer)
-	// duration := parseDuration(self.buffer)
+	value, shardPartition, partitionlessQuery := parseValuesFromBuffer(self.buffer)
 
 	log := &PostgresLogLine{
-		Timestamp:     timestamp,
-		Username:      user,
-		Database:      database,
-		Duration:      duration,
-		LogType:       logType,
-		StatementName: statementName,
-		Value:         value,
+		Timestamp:          timestamp,
+		Username:           user,
+		Database:           database,
+		Duration:           duration,
+		LogType:            logType,
+		StatementName:      statementName,
+		Value:              value,
+		ShardPartition:     shardPartition,
```

Comment:
This seems a little strange to add to the postgres log line parser. It's really only supposed to know how to parse a log line. We should instead look at adding this to a new struct. Maybe as a part of the slow query logging method? https://github.com/mxenabled/timber/blob/9f553bf9125c2a3a597ddacdf43572a0deec0025/slow_query.go#L24-L43

Reply:
Thats not the updated file.

Reply:
Nope, it's from the `main` branch.